src/WebJobs.Extensions.DurableTask/DurableTaskJobHostConfigurationExtensions.cs

```diff
@@ -4,6 +4,7 @@
 using System;
 using System.Net.Http;
 using System.Threading;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask.Correlation;
```

Comment:
I think this namespace is unused if Functions V1, so move it down a line so it is only used in non-Functions V1 compilations.

Reply:
Done